src/lua/httpc.lua

```diff
@@ -32,8 +32,18 @@
 local driver = package.loaded.http.client
```

Comment:
Regarding the commit message (1st patch):

> but content type is passed to encoder function without options [1].

As I see, the value of the header is passed to the function in current version of the patch.